### PR TITLE
Add missing nullable and nonnull annotations.  Only log attribute not found if it is not found

### PR DESCRIPTION
**Pull request by thomaszurkan-optimizely**

### CODE REVIEW

**Review by mikeproeng37 — APPROVED** [review]
lgtm

**Comment by thomaszurkan-optimizely** [comment]
build

File: Pods/Pods.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -4188,9 +4177,8 @@
 				SDKROOT = iphoneos;
 				SKIP_INSTALL = YES;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = "$(inherited) ";
-				SWIFT_COMPILATION_MODE = wholemodule;
-				SWIFT_OPTIMIZATION_LEVEL = "-O";
-				SWIFT_VERSION = 4.2;
+				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
```
Comment (msohailhussain) [inline Pods/Pods.xcodeproj/project.pbxproj:4180]: what does -O means?

File: Pods/Pods.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -4188,9 +4177,8 @@
...
+				SWIFT_VERSION = 4.0;
```
Comment (msohailhussain) [inline Pods/Pods.xcodeproj/project.pbxproj:4181]: it was targeted before 4.2, now targeting 4.0 ?

**Comment by msohailhussain** [comment]
lgtm
swift 4.0 is targeted before that it was 4.2. from the commit 'mixpanel' doesn't support 4.2, is it the reason for it?

**Review by msohailhussain — APPROVED** [review]

**Pull request merged**